### DISCUSSION TITLE
I try to get github student pack can't

### DISCUSSION THREAD

**Discussion by GamilaCoding** [community/community#53270] [GitHub Education]
I am on the Frontend masters website and trying to get a GitHub student pack and didn't works two times any help?
![req](https://user-images.githubusercontent.com/123306614/233248711-bce9d28f-14e0-4c8f-8680-172301140a80.JPG)

**Comment by samus-aran** [comment]
Hi @mermaidseaweav 

If your school isn't a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution then we would recommend informal schools to apply first to the [GitHub Campus program](https://education.github.com/schools) to be verified.

**Reply by GamilaCoding** [reply]
but frontend masters in lists of schools here but didn't show to me?
and can you give me more details for screenshots (jpg) to accepted enroll because I try a lot didn't accepted
